docs/DeveloperGuide.md

```diff
@@ -910,25 +961,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **Extensions**
 
-* 2a. The list is empty.
+* 1a. The list is empty.
 
   Use case ends.
 
-* 4a. The social link is invalid.
+* 3a. The social link is invalid.
 
-    * 4a1. The user will see the error page displayed by the browser used to load the page link.
-
-      Use case ends.
-
-* 4b. The page is valid but fails to load.
-
-    * 4b1. The user will see the error page displayed by the browser used to load the page link.
+    * 3a1. The user will see the error page displayed by the browser used to load the page link.
 
       Use case ends.
 
-* 4c. The social link directs to a social media app which the user has not logged in.
+* 3b. The page is valid but fails to load.
 
-    * 4c1. The user will be taken to the sign-in page of the social media app.
+    * 3b1. The user will see the error page displayed by the browser used to load the page link.
 
```

Comment:
Same comments as the `email` command above.

Reply:
This happens to any link. As for the email app, we can prefill with the target email address, but for the social link, the authentication logic is complete handled by the browser, so it is not worth mentioning here.